doc/user/content/transform-data/optimization.md

```diff
@@ -49,20 +49,20 @@ EXPLAIN SELECT * FROM foo WHERE x = 42 AND y = 'hello';
      ReadExistingIndex materialize.public.foo_x_y lookup_value=(42, "hello")+
                                                                             +
  Used Indexes:                                                              +
-   - materialize.public.foo_x_y                                             +
+   - materialize.public.foo_x_y (lookup)                                    +
 ```
 
 #### Matching multi-column indexes to multi-column `WHERE` clauses
 
 In general, your index key should exactly match the columns that are constrained in the `WHERE` clause. In more detail:
-- If the `WHERE` clause constrains fewer fields than your index key includes, then **the index will not be used**. For example, an index on `(x, y)` cannot be used to speed up `WHERE x = 7`.
+- **_If the `WHERE` clause constrains fewer fields than your index key includes, then the index will not be used_** for a lookup, but will be fully scanned. For example, an index on `(x, y)` cannot be used to execute `WHERE x = 7` as a point lookup.
```

Comment:
nit: you are doing an emphasis in a combination of bold and italic, which in general is an anti-pattern. cc @parkerhendo for your opinion as the expert on fonts and layouts.

Also, I suggest to highlight either the entire sentence or only this bit:

> If the `WHERE` clause constrains fewer fields than your index key includes, [...]

Reply:
Other documents use the

```text
{{< warning >}}
Some highlighted text
{{< warning >}}
```

convention to highlight important information, but let's see what @morsapaes has to say about this.

Reply:
Thanks, I changed it to `{{< warning >}}` for now. Let's see what @morsapaes will say.

Reply:
I'm not sure if we have an existing style guide or rules for this, but I'd lean on **bolding** words or phrases you want to emphasize. I'd personally recommend against emphasizing entire sentences, but that's totally subjective, so feel free to ignore.

Reply:
Thanks! I ended up doing it with `{{< warning >}}`.